GherkinSyncTool.Synchronizers.TestRail/Model/PriorityId.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace GherkinSyncTool.Synchronizers.TestRail.Model
 {
-    public enum PriorityId : ulong
+    public enum PriorityId : byte
```

Comment:
We can leave it as default type, without explicit statement - no need to use `byte` here

Reply:
Fixed